src/pages/atomicdex/api/v20/index.mdx

```diff
@@ -106,9 +106,155 @@ It includes a uniform request, successful and error response formats. At the mom
+| ------------------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
+| required_confirmations    | integer   | Optional. Confirmations to wait for steps in swap. Defaults to value in the coins file if not set.                                               |
+| requires_notarization     | boolean   | Optional, defaults to `false`. For [dPoW](https://komodoplatform.com/en/blog/dpow-demystified/) protected coins, a `true` value will wait for transactions to be notarised when doing swaps. Overrides value if set in `coins` file. |
+| priv_key_policy           | string    | Defaults to `ContextPrivkey`. Set as `Trezor` to activate in Trezor mode.                                                                                                                                        |
+| min_addresses_number      | integer   | HD wallets only. How many additional addreesses to generate at a minimum.                                                                                                                                                         |
+| scan_policy               | string    | HD wallets only. Whether or not to scan for new addresses. Select from `do_not_scan`, `scan_if_new_wallet` or `scan`. Note that `scan` will result in multple requests to the Komodo DeFi SDK.                                      |
+| gap_limit                 | integer   | HD wallets only. The max number of empty addresses in a row. If transactions were sent to an address outside the `gap_limit`, they will not be identified when scanning.                                                          |
+| zcash_params_path         | string    | ZHTLC coins only. Path to folder containing [Zcash parameters](https://z.cash/technology/paramgen/). Optional, defaults to standard location as defined in [this guide](https://forum.komodoplatform.com/t/installing-zcash-params/603)                |
+| scan_blocks_per_iteration | integer   | ZHTLC coins only. Sets the number of scanned blocks per iteration during `BuildingWalletDb` state. Optional, default value is 1000.                                                                                                                    |
+| scan_interval_ms          | integer   | ZHTLC coins only. Sets the interval in milliseconds between iterations of `BuildingWalletDb` state. Optional, default value is 0.                                                                                                                      |
+| sync_params               | integer or string | ZHTLC coins only. Optional, defaults to two days ago. Defines where to start scanning blockchain data upon initial activation. Options: `"earliest"` (the coin's sapling_activation_height), `height` (a specific block height) or `date` (a linux timestamp).            |
```

Comment:
I think the term `a unix timestamp` sounds cooler than linux.

Reply:
agree - done